testutil/base.go

```diff
@@ -56,7 +57,36 @@ func (s *BaseTest) TearDownTest(c *check.C) {
 	s.cleanupHandlers = nil
 }
 
-// AddCleanup adds a new cleanup function to the test
+// AddCleanup adds a new cleanup function to the test.
 func (s *BaseTest) AddCleanup(f func()) {
 	s.cleanupHandlers = append(s.cleanupHandlers, f)
 }
+
+// Backup the specified list of elements before further mocking.
```

Comment:
now that the BaseTest is not changed anymore, this placement is a bit not optiomal but can be changed in a follow up